### PR TITLE
Add tests for pathToJestPackageJSON

### PR DESCRIPTION
**Pull request by seanpoulter**
Adding some tests for #218 before looking at #222.

### CODE REVIEW

File: tests/helpers.test.ts [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+import { pathToJestPackageJSON } from '../src/helpers'
+import { existsSync } from 'fs'
+import * as path from 'path'
+
+const defaultPathToJest = 'node_modules/.bin/jest'
+
+describe('ModuleHelpers', () => {
+  describe('pathToJestPackageJSON', () => {
+    it('should return null when not found', () => {
+      ;(existsSync as jest.Mock<boolean>).mockReturnValueOnce(false).mockReturnValueOnce(false)
```
Comment (orta) [inline tests/helpers.test.ts]: I'd take `;(existsSync as jest.Mock<boolean>)` and move it to line for 

`const existsMock = existsSync as jest.Mock<boolean>` or `const existsMock = existsSync as any as jest.Mock<boolean>` if you have to

Comment (seanpoulter) [reply tests/helpers.test.ts]: That sounds good. 👍

**Comment by seanpoulter** [comment]
Back to you!

**Comment by orta** [comment]
👍

**Comment by seanpoulter** [comment]
Thanks @orta.

**Pull request merged**